backend/isisInternal.py

```diff
@@ -208,6 +208,22 @@ def getGoToCycle(instrument, search):
     print(endTime - startTime)
     return "Not Found"
 
+# Get spectra data
+
+
+@app.route('/getSpectra/<instrument>/<cycle>/<runs>/<spectra>')
+def getSpectra(instrument, cycle, runs, spectra):
+    data = nexusInteraction.spectra(instrument, cycle, runs, spectra)
+    return jsonify(data)
```

Comment:
If this function just returns a single spectrum then it should be called `getSpectrum`.